common/app/views/fragments/inlineSvg.scala.html

```diff
@@ -1,4 +1,4 @@
-@(file: String, path: String = "", classes: Seq[String] = List())
-<span class="inline-@file @if(path){inline-@path} @classes.mkString(" ")">
+@(file: String, path: String = "", classes: Seq[String] = List(), attributes: Map[String, String] = Map())
```

Comment:
it feels like there's a good argument for just allowing a string for further classes/attribtues. makes it clear to read and seems far less engineered?